packages/serverless/settings/observability_project/index.ts

```diff
@@ -11,4 +11,20 @@ import * as settings from '@kbn/management-settings-ids';
 export const OBSERVABILITY_PROJECT_SETTINGS = [
   settings.ML_ANOMALY_DETECTION_RESULTS_ENABLE_TIME_DEFAULTS_ID,
   settings.ML_ANOMALY_DETECTION_RESULTS_TIME_DEFAULTS_ID,
+  settings.OBSERVABILITY_ENABLE_COMPARISON_BY_DEFAULT_ID,
+  settings.OBSERVABILITY_APM_DEFAULT_SERVICE_ENVIRONMENT_ID,
+  settings.OBSERVABILITY_APM_SERVICE_GROUP_MAX_NUMBER_OF_SERVCIE_ID,
+  settings.OBSERVABILITY_ENABLE_INSPECT_ES_QUERIES_ID,
+  settings.OBSERVABILITY_APM_LABS_BUTTON_ID,
+  settings.OBSERVABILITY_APM_AWS_LAMBDA_PRICE_FACTOR_ID,
+  settings.OBSERVABILITY_APM_AWS_LAMBDA_REQUEST_COST_PER_MILLION_ID,
+  settings.OBSERVABILITY_APM_ENABLE_SERVICE_METRICS_ID,
+  settings.OBSERVABILITY_APM_ENABLE_CONTINUOUS_ROLLUPS_ID,
+  settings.OBSERVABILITY_APM_AGENT_EXPLORER_VIEW_ID,
+  settings.OBSERVABILITY_APM_ENABLE_PROFILING_INTEGRATION_ID,
+  settings.OBSERVABILITY_APM_PROGRESSIVE_LOADING_ID,
```

Comment:
It seems the settings on lines 25-29 are not displayed in APM General settings UI, so why do we want to enable them? Are they needed for any other functionality in the Serverless observability project?

Reply:
They are needed for the `Labs` feature, you can see the settings that are available there in the second video (starting at 0:33).

<img width="797" alt="image" src="https://github.com/elastic/kibana/assets/5831975/c40f6540-1ad3-4815-8a19-a9bab15b3130">

Reply:
Got it, thanks for clarifying!